packages/app/components/claim/claim-button.tsx

```diff
@@ -221,12 +252,79 @@ export const ClaimButton = ({
             {isAuthenticated ? "Save to Collect" : "Save on Spotify"}
           </Text>
         </>
-      );
-    } else if (
-      edition?.gating_type === "spotify_presave" ||
-      edition?.gating_type === "music_presave"
-    ) {
-      return (
+      </Button>
+    );
+  } else if (edition?.gating_type === "multi_provider_music_save") {
+    return (
+      <View tw="w-full flex-row">
+        {edition?.apple_music_track_url ? (
+          <Button
+            {...buttonProps}
+            onPress={() => handleCollectPress("appleMusic")}
+            tw="flex-1 flex-row items-center justify-center"
+            disabled={isAppleMusicCollectLoading}
+          >
+            <AppleMusic
+              width={22}
```

Comment:
I didn't check this icon on the website, but I feel that its size may be incorrect because it appears to lack spacing. I'll check it!
- Spotify 
![CleanShot 2023-04-25 at 12 45 32](https://user-images.githubusercontent.com/37520667/234062513-6e87b705-5aad-4499-b385-3a0f71798049.png)

- Apple 
![CleanShot 2023-04-25 at 12 44 17](https://user-images.githubusercontent.com/37520667/234062365-123b995f-ff7b-4e7c-a57e-2755d59bf52d.png)